docs/zopebook/SearchingZCatalog.rst

```diff
@@ -25,110 +25,115 @@ Automatic Cataloging
 Getting started with Mass Cataloging
 ------------------------------------
 
-Let's take a look at how to use the ZCatalog to search documents.
+Let's take a look at how to use the **ZCatalog** to search documents.
 Cataloging a bunch of objects all at once is called *mass cataloging*.
+
 Mass cataloging involves four steps:
 
-- Creating a ZCatalog
+- Creating a **ZCatalog**
 
 - Creating indexes
 
 - Finding objects and cataloging them
 
-- Creating a web interface to search the ZCatalog.
+- Creating a web interface to search the **ZCatalog**
 
 Creating a ZCatalog
 -------------------
 
-Choose *ZCatalog* from the product add list to create a ZCatalog
-object within a subfolder named 'Zoo'.  This takes you to the
-ZCatalog add form, as shown in the figure below.
+Choose ``ZCatalog`` from the product add list to create a **ZCatalog**
+object within a subfolder named ``Zoo``.  This takes you to the
+**Add ZCatalog** form, as shown in the figure below.
 
 .. figure:: Figures/creatingzcatalog.png
 
-   ZCatalog add form
-
-The Add form asks you for an *Id* and a *Title*.  Give your
-ZCatalog the Id 'AnimalCatalog' and click *Add* to create your new
-ZCatalog.  The ZCatalog icon looks like a folder with a small
-magnifying glass on it.  Select the *AnimalCatalog* icon to see
-the *Contents* view of the ZCatalog.
-
-A ZCatalog looks a lot like a folder, but it has a few more
-tabs.  Six tabs on the ZCatalog are the exact same six tabs you
-find on a standard folder.  ZCatalog have the following views:
-*Contents*, *Catalog*, *Properties*, *Indexes*, *Metadata*,
-*Find Objects*, *Advanced*, *Undo*, *Security*, and *Ownership*.
-When you click on a ZCatalog, you are on the *Contents*
-view. Here, you can add new objects and the ZCatalog will
-contain them just as any folder does. Although a ZCatalog is
+   Add ZCatalog form
+
+The Add form asks you for an **Id** and a **Title**.  Give your
+**ZCatalog** the **Id** ``AnimalCatalog`` and click **Add** to create your new
+**ZCatalog**.  The **ZCatalog** icon looks like a folder with a small

```

Comment:
Nowadays with Bootstrap ZMI the folder is gone, the icon is only a magnifying glass.